.eslintrc

```diff
@@ -1,2 +1,4 @@
 # We use JavaScript Standard Style (https://github.com/feross/standard)
 extends: standard
+rules:
```

Comment:
Looks like Alan already took care of it though: https://github.com/interledger/five-bells-ledger/pull/190/files